lib/transformations/resolve/resolve.js

```diff
@@ -0,0 +1,46 @@
+// Press ctrl+space for code completion
+export default function transformer(file, api) {
```

Comment:
Yeah. That is a concern. I've better understood this now. So I will update the issue with better info by the end of this week!
Also lets Dm on slack if you have any specific questions 👍